### DISCUSSION TITLE
想问下steam++在同步到gitee的时候，release是手动添加的还是跟着github仓库一起同步过去的

### DISCUSSION THREAD

**Discussion by swift-lee** [BeyondDimension/SteamTools#338] [Q&A]
想建个镜像仓库，结果gitee下载tar.gz要机器人验证，想用release绕过去结果release死活不能自动同步过去
实不相瞒，同步的action我直接用了这个仓库里的🤣

**Comment by rmbadmin** [comment]
release确实不会自动同步，release只能用CI尝试自动化发布，但gitee那边我也不知道能否实现。
